tests/data/XmlValidation/valid_Test.xml

```diff
@@ -2,6 +2,7 @@
 <Site BuildName="dummy_test_name"
   BuildStamp="dummy_stamp"
   Name="dummy_name"
+  ModelName="dummy_model"
```

Comment:
Since this is a real field we expect to add to CDash eventually, perhaps it would be better to use an attribute name we never expect to ever see as part of the test?